improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -303,16 +281,9 @@ def test_radii_varying_with_lead_time_check_data(self):
         varies with lead time and that a cube is returned.
```

Comment:
I've changed this test to only contain one leadtime rather than 3, but otherwise kept the test the same